templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,237 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-hello-simple">
+    <style>
+      #{{ snippet_id }}-ffos-circling-fox {
+          /* All CSS should be preceeded by the ID selector for this individual
+            snippet. */
+      }
+    </style>
+    
+    {% if icon_url %}<a href="#" id="hello-cta">{% endif %}
+    <img class="icon" src="{{ icon }}" />
+    {% if icon_url %}</a>{% endif %}
+    <p>{{ text|safe }}</p>
+
+    <script type="text/javascript">
+      //<![CDATA[
+      var TARGET_AVAILABLE;
+      var TARGET_NAME = 'loop';
+      var INELIGIBLE_VIEW = 'hello-ineligible-view';
+      var INELIGIBLE_CLICK = 'hello-ineligible-click';
+      var ELIGIBLE_VIEW = 'hello-eligible-view';
+      var ELIGIBLE_CLICK = 'hello-open';
+      var START_CONVERSATION = 'hello-start-conversation';
+      var URL_COPIED = 'hello-url-copied';
+      var URL_EMAILED = 'hello-url-emailed';
+      var PAGE_ID_SOURCE = 'hello-tour_OpenPanel_snippet-{{ snippet_id }}';
+
+      (function() {
+          var snippet = document.getElementById('{{ snippet_id }}-hello-simple');
+          snippet.addEventListener('show_snippet', function() {
+              TARGET_AVAILABLE = Mozilla.UITour.isTargetAvailable(TARGET_NAME, true);
+          }, false);
+      })();
+
+      // create namespace
+      if (typeof Mozilla == 'undefined') {
+        var Mozilla = {};
+      }
+
+      ;(function($) {
+        'use strict';
+
+        // create namespace
```

Comment:
We don't need the entire UITour library included. Take, for example, https://github.com/mozilla/snippets-service/blob/79eaa9aef4caca9329af5780aa1374019e78916c/snippets/base/templates/base/includes/snippet_js.html#L314, where we just create the custom event directly instead of including or relying on the UITour library.

You should be able to create the custom events and add the event listeners you need directly. It'll be less code and easier to read.